cogs/assistance.py

```diff
@@ -286,7 +286,7 @@ async def updateb9s(self, ctx):
         embed.description = "A guide for updating to new B9S versions."
         await ctx.send(embed=embed)
 
-    @commands.command()
+    @commands.command(name = "updateluma", aliases = ["lumaupdate"])
```

Comment:
Why the  `name = "updateluma"`?

Reply:
Because that’s what the original name was. It’s the guide for manually updating luma, and there are a bunch of instances where I’ve found myself and others typing lumaupdate instead. Also it doesn’t really affect anything so i figured why not make a PR

Reply:
Setting name isn't needed because the method name is also updateluma.

Reply:
ok, im not actually sure how to change that in the pull request.